python/cugraph/cugraph/dask/community/egonet.py

```diff
@@ -110,6 +111,15 @@ def ego_graph(input_graph, n, radius=1, center=True):
     # Initialize dask client
     client = input_graph._client
 
+    # FIXME: Implement a better way to check if the graph is weighted similar
+    # to 'simpleGraph'
+    if len(input_graph.edgelist.edgelist_df.columns) != 3:
```

Comment:
C++ egonet does not require weights.  Not sure we should require that in python.

Reply:
I get a `segmentation fault` when running the CAPI tests